### PR TITLE
Solves #36 Added Guide to start development of web app on your local host.

### PR DESCRIPTION
**Pull request by shobhitexe**
Solves #36 

Hey, have added the
- Tech Stack 
- Prerequisites
- Additional resources
- Steps To setup StarterHive Locally
- Steps to make a contribution
- dos and don'ts to keep in mind when creating a pull request

Anything else to add Plz review it

### CODE REVIEW

**Comment by shobhitexe** [comment]
Solves #36

File: web/README.md [hunk 1]
```diff
@@ -1,4 +1,113 @@
...
+  <li>Do follow the project's coding style and guidelines.</li>
+  <li>Do break down significant changes into smaller, manageable commits.</li>
+  <li>Do test your changes thoroughly before submitting the pull request.</li>
+  <li>Do respond to any feedback or comments promptly and respectfully.</li>
+  <li>Do make sure your code builds and runs without errors.</li>
+  <li>Do document any necessary changes to the project's documentation or README files.</li>
+  <li>Do consider the impact of your changes on other parts of the codebase.</li>
+</ul>
+
+### Don'ts:
```
Comment (ArslanYM) [inline web/README.md:102]: Remove these.

File: web/README.md [hunk 2]
```diff
@@ -1,4 +1,113 @@
...
+![image](https://github.com/shobhitexe/StarterHive/assets/29349136/1fde1c2b-b64e-4fc5-b9b1-0363290a1900)
+##### Step 5.2: When you click on that button, you will find an 'Open pull request' button. Click on it to create a pull request
+![image](https://github.com/shobhitexe/StarterHive/assets/29349136/fbad9a46-275d-435d-bc7c-88a93d9aafb1)
+##### Step 5.3: Now, on this page, write a good and descriptive title and comment. Click on the 'Create pull request' button to submit a pull request
+![image](https://github.com/shobhitexe/StarterHive/assets/29349136/031a92e1-b626-47d0-863f-37409ccf2c33)
+
+## Tada! 🥳 You have just created your first pull request. Once the maintainers review and accept it, your contribution will be merged.
+
+## Here are some dos and don'ts to keep in mind when creating a pull request (PR):
+### Dos:
```
Comment (ArslanYM) [inline web/README.md:88]: Remove these as well.

File: web/README.md [hunk 3]
```diff
@@ -1,4 +1,113 @@
-# Provide a guide for development on local host 
-- Tech used
- - Vite and tailwind css 
- 
\ No newline at end of file
+# # Setting up and Running Locally and Contributing
```
Comment (ArslanYM) [inline web/README.md:1]: Only one # should be used here

File: web/README.md [hunk 4]
```diff
@@ -1,4 +1,113 @@
...
+- [React.Js Documentation](https://react.dev/)
+- [Tailwind Documentation](https://tailwindcss.com/docs/installation)
+- [Vite Documentation](https://vitejs.dev/guide/)
+- [Git Documentation](https://git-scm.com/doc)
+- [MDN Web Docs](https://developer.mozilla.org/en-US/) (A comprehensive resource for web developers)
+ 
+
+## To setup StarterHive Locally 
+
+### Step 1: Fork the repository
```
Comment (ArslanYM) [inline web/README.md:28]: no need to provide images as of now. Just provide link to guides on forking etc from the official github guide. 
Link : https://docs.github.com/en/get-started/quickstart

**Comment by shobhitexe** [comment]
> @shobhitexe When contributing to an open source project, Try to keep your commits and PRs as clean as possible so its easy for maintainers to merge your PR. As of now this PR looks full of bugs and mistakes. I would suggest closing this PR, making changes as said. And then creating a new PR. Thankyou.

Sure got it

**Pull request closed without merging**